<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadlevelerMonitor::Widgets::JobDetailWidget</class>
 <widget class="QWidget" name="LoadlevelerMonitor::Widgets::JobDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loadleveler Job Detail Viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="text">
        <string>Job ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="job_id_line_edit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="style_group_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QToolButton" name="tree_style_button">
          <property name="toolTip">
           <string>表格</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../loadleveler_monitor.qrc">
            <normaloff>:/loadleveler_monitor/images/table-style.png</normaloff>:/loadleveler_monitor/images/table-style.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="response_style_button">
          <property name="toolTip">
           <string>图形</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../loadleveler_monitor.qrc">
            <normaloff>:/loadleveler_monitor/images/response-style.png</normaloff>:/loadleveler_monitor/images/response-style.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="output_style_button">
          <property name="toolTip">
           <string>文本</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../loadleveler_monitor.qrc">
            <normaloff>:/loadleveler_monitor/images/text-style.png</normaloff>:/loadleveler_monitor/images/text-style.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="stacked_widget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="output_style_page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="output_text_browser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tree_style_page">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTreeView" name="tree_view"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="response_style_page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="response_text_browser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_output_style">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../loadleveler_monitor.qrc">
     <normaloff>:/loadleveler_monitor/images/text-style.png</normaloff>:/loadleveler_monitor/images/text-style.png</iconset>
   </property>
   <property name="text">
    <string>Output style</string>
   </property>
  </action>
  <action name="action_tree_style">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../loadleveler_monitor.qrc">
     <normaloff>:/loadleveler_monitor/images/table-style.png</normaloff>:/loadleveler_monitor/images/table-style.png</iconset>
   </property>
   <property name="text">
    <string>Tree style</string>
   </property>
  </action>
  <action name="action_response_style">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../loadleveler_monitor.qrc">
     <normaloff>:/loadleveler_monitor/images/response-style.png</normaloff>:/loadleveler_monitor/images/response-style.png</iconset>
   </property>
   <property name="text">
    <string>Response style</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../loadleveler_monitor.qrc"/>
 </resources>
 <connections/>
</ui>
